Tests/Tests/STPSourceTest.m

```diff
@@ -111,12 +33,11 @@ - (void)testDecodingSource_ideal {
     XCTAssertEqual(source.status, STPSourceStatusPending);
     XCTAssertEqual(source.type, STPSourceTypeIDEAL);
     XCTAssertEqual(source.usage, STPSourceUsageSingleUse);
-    XCTAssertEqualObjects(source.details, response[@"ideal"]);
+    XCTAssertEqualObjects(source.details[@"bank"], @"ing");
```

Comment:
Is it strange that the tests here are checking the decoding that is all happening inside STPFixtures? Should all the decoding tests be moved into like STPFixturesTest or something?